.nvmrc

```diff
@@ -0,0 +1 @@
+12
```

Comment:
can you add this to the .gitignore please?

Reply:
Actually let me delete this file, i was not using it correctly.